app/controllers/actions/other/DescriptionRequest.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.actions.other
+
+import models.UserAnswers
+import models.requests.DataRequest
+import play.api.mvc.WrappedRequest
+
+case class DescriptionRequest[T](request: DataRequest[T], description: String) extends WrappedRequest[T](request){
+  val userAnswers:UserAnswers = request.userAnswers
+}
```

Comment:
There is one of these in class of beneficiary actions. Can one of them be moved to controllers/actions and the other be removed to avoid duplication (similar to BeneficiaryNameRequest)?

Reply:
Done